.circleci/config.yml

```diff
@@ -725,7 +725,18 @@ jobs:
     steps:
       - checkout
       - *restore_cache_step
-      - run: scripts/run-tox-scenario '^algoliasearch_contrib-'
+      - run: scripts/run-tox-scenario '^algoliasearch_contrib-.*-algoliasearch$'
+      - *persist_to_workspace_step
+      - *save_cache_step
+
+  algoliasearch2:
```

Comment:
We don't need a separate job for this, we can do it all in one `^algoliasearch_contrib-*` is fine.